stylesheets/_component.block-list.scss

```diff
@@ -101,6 +101,10 @@ $width: 500px;
     }
 }
 
+.block-list__item.is-disabled {
```

Comment:
Not sure if this is a relevant comment, but this feels half BEM half not (which may not be a problem) - I'd expect this to add a modifier a long the lines of `block-list__item--disabled`